examples/gwlb_with_vmseries/example.tfvars

```diff
@@ -0,0 +1,281 @@
+      "application_inbound" = {
+        name = "application-inbound-nsg"
+        rules = {
+          app_inbound = {
+            name                       = "application-allow-inbound"
+            priority                   = 100
+            direction                  = "Inbound"
+            access                     = "Allow"
+            protocol                   = "Tcp"
+            source_address_prefixes    = ["134.238.135.14", "134.238.135.140"]
+            source_port_range          = "*"
+            destination_address_prefix = "*"
+            destination_port_ranges    = ["22", "80", "443"]
+          }
+        }
+      }
+    }
+    subnets = {
+      "app1" = {
+        name                       = "app1-snet"
+        address_prefixes           = ["10.0.2.0/28"]
+        network_security_group_key = "application_inbound"
+      }
+    }
+  }
+}
+
+
+# --- LOAD BALANCING PART --- #
+load_balancers = {
+  "app1" = {
+    name = "app1-lb"
+    nsg_auto_rules_settings = {
+      nsg_vnet_key = "app1"
+      nsg_key      = "application_inbound"
+      source_ips   = ["0.0.0.0/0"]
+    }
+    frontend_ips = {
+      "app1" = {
+        name             = "app1"
+        public_ip_name   = "public-lb-app1-pip"
+        create_public_ip = true
+        gwlb_key         = "gwlb"
+        in_rules = {
+          "balanceHttp" = {
+            name        = "HTTP"
+            protocol    = "Tcp"
+            port        = 80
+            floating_ip = false
+          }
+          "balanceHttps" = {
+            name        = "HTTPS"
+            protocol    = "Tcp"
+            port        = 443
+            floating_ip = false
+          }
+        }
+        out_rules = {
+          outbound = {
+            name     = "tcp-outbound"
+            protocol = "Tcp"
+          }
+        }
+      }
+    }
+  }
+}
+
+# --- GWLB PART --- #
+gateway_load_balancers = {
+  gwlb = {
+    name = "vmseries-gwlb"
+
+    frontend_ip = {
+      vnet_key   = "transit"
+      subnet_key = "data"
+    }
+
+    health_probe = {
+      name     = "custom-health-probe"
+      port     = 80
+      protocol = "Tcp"
+    }
+
+    backends = {
+      backend = {
+        name = "custom-backend"
+        tunnel_interfaces = {
+          internal = {
+            identifier = 800
+            port       = 2000
+            protocol   = "VXLAN"
+            type       = "Internal"
+          }
+          external = {
+            identifier = 801
+            port       = 2001
+            protocol   = "VXLAN"
+            type       = "External"
+          }
+        }
+      }
+    }
+
+    lb_rule = {
+      name = "custom-lb-rule"
+    }
+  }
+}
+
+# --- VMSERIES PART --- #
+bootstrap_storages = {
+  "bootstrap" = {
+    name = "examplegwlbbootstrap"
+    storage_network_security = {
+      vnet_key            = "transit"
+      allowed_subnet_keys = ["management"]
+      allowed_public_ips  = ["134.238.135.14", "134.238.135.140"]
```

Comment:
same here

Reply:
Updated. 

Nevertheless in other examples we have the same IPs: https://github.com/PaloAltoNetworks/terraform-azurerm-swfw-modules/blob/5e8e2660b79d9adef45261abe9ba456dfc64bf6d/examples/dedicated_vmseries/example.tfvars#L25